### PR TITLE
Added a custom Option type and extended the select and added a few new classes for typography

### PR DESCRIPTION
**Pull request by SilasMarvin**

### CODE REVIEW

File: pgml-dashboard/static/css/scss/base/_typography.scss [hunk 1]
```diff
@@ -11,10 +13,60 @@
...
-.text-black {color: #{$gray-900} !important;}
-.text-white {color: #{$gray-100} !important;}
-.text-soft-white { color: #{$gray-200} !important; }
+.text-black {
+    color: #{$gray-900} !important;
+}
+.text-white {
+    color: #{$gray-100} !important;
+}
+.text-gradient-blue {
```
Comment (chillenberger) [inline pgml-dashboard/static/css/scss/base/_typography.scss:35]: Using a mixin for this would prevent the repetition.

**Review by chillenberger — APPROVED** [review]

File: pgml-dashboard/static/css/scss/base/_typography.scss (hunk 1 above)
Comment (SilasMarvin) [reply pgml-dashboard/static/css/scss/base/_typography.scss:35]: Let me update this. I haven't used mixins before

Comment (levkk) [reply pgml-dashboard/static/css/scss/base/_typography.scss]: classes are better than mixins, mixings generate a ton of repetitive css code, while classes inherit and cascade.

Comment (chillenberger) [reply pgml-dashboard/static/css/scss/base/_typography.scss]: @extend is also an option that minimizes css code (same as having two classes) and reduces the number of classes an element needs to take.  If we do go with two classes here we should have the _typographic class .text-gradient then a utility class for all the gradients so they can be applies as text color, background color, border color ect.

**Pull request merged**